### DISCUSSION TITLE
Invalid preference category path: Birt 4.9.0

### DISCUSSION THREAD

**Discussion by jmenzies12** [eclipse-birt/birt#1391] [Q&A] [answered]
Hey there,

We are getting a message in our console in regards to invalid preference category paths:

!MESSAGE Invalid preference category path: org.eclipse.birt.chart.reportitem.ui.ChartPreferencePage (bundle: org.eclipse.birt.chart.ui.extension, page: org.eclipse.birt.chart.ui.swt.fieldassist.preferences.FieldAssistPreferencePage)

Is this something that has been fixed in a newer version of Birt?

Cheers
Jonthan

**Comment by speckyspooky** [comment, marked as answer]
There are many changes and fixes implemented since version BIRT4.9.0
But I cannot say if your issue was fixed because the message is completely new for me.

Therefore it would be great if you could test try it on your side with the latest 4.14.0 built version.
If it is not fixed you can open an issue ticket with an simple demo-report that a research with the demo can be done.
